tests/server/test_mock_power.py

```diff
@@ -0,0 +1,121 @@
+# Copyright 2016 Cisco Systems, Inc.
+#
```

Comment:
@britthouser yet to go through the test itself.. but I think we should create a new hierarchy for unit-tests.. `tests/unit-tests/*`. so this particular file should be `tests/unit-tests/server/power.py`. Anything under `unit-tests` should never need a live connection.. let us do away with test or mock in the file name too.